src/scripts/achievements/AchievementHandler.ts

```diff
@@ -391,13 +391,21 @@ class AchievementHandler {
         /*
          * REGIONAL
          */
-        const addGymAchievements = (gyms: string[], category: GameConstants.Region | GameConstants.ExtraAchievementCategories) => {
+        const addGymAchievements = (gyms: string[], category: GameConstants.Region | GameConstants.ExtraAchievementCategories, subregion?: string) => {
             gyms.forEach(gym => {
-                const gymTitle: string = gym.includes('Elite') || gym.includes('Champion') ? gym : `${gym} Gym`;
+                const displayName = GymList[gym]?.displayName;
+
+                // Name of person's title if elite/champion, else the gym's town name
+                const gymTitle: string = gym.includes('Elite') || gym.includes('Champion') || gym.includes('Gym') ? gym.replace(/\d/g, '') : `${gym.replace(/\d/g, '')} Gym`;
```

Comment:
This solution is most likely breaking if the player tries to track for example a Bruno achievement. We will have 2 achievements with the same name, and then we don't know which one to track, when we open a save